PayPalWebCheckout/src/main/java/com/paypal/android/checkoutweb/PayPalWebCheckoutListener.kt

```diff
@@ -0,0 +1,26 @@
+package com.paypal.android.checkoutweb
+
+import com.paypal.android.core.PayPalSDKError
+
+/**
+ * Implement this callback to receive results from [PayPalWebCheckoutClient].
+ */
+interface PayPalWebCheckoutListener {
+
+    /**
+     * Called when the PayPal flow completes successfully.
+     * @param result [PayPalWebCheckoutResult] with order information.
+     */
+    fun onPayPalWebSuccess(result: PayPalWebCheckoutResult)
```

Comment:
Nitpick but let's use `WebCheckout` for everything - so `onPayPalWebCheckoutSuccess`, `onPayPalWebCheckoutFailure` ... unless you all think that's too long

Reply:
@sarahkoop im kinda worried about redundancy. I think that the context for the merchant is sufficient. Like that interface can only be used for the flow, the interface name specifys it, and also the `paypalweb` prefix on the methods. It also could be annoying for merchants when they are developing, like the `autocomplete`, they would have to write the prefix all the way to get to the class they want. Bit of a nitpick though hehe

Reply:
Yeah that makes sense ok good with me 👍